### PR TITLE
metrics: collect extra data in jenkins assets.

### PR DESCRIPTION
**Pull request by jcvenegas**
Generate report and store json data from teardown.sh

Fixes: #2659

Signed-off-by: Jose Carlos Venegas Munoz <jose.carlos.venegas.munoz@intel.com>

### CODE REVIEW

File: .ci/teardown.sh [hunk 1]
```diff
@@ -68,6 +68,10 @@ collect_logs()
 	local -r virtiofs_log_path="${log_copy_dest}/${virtiofs_log_filename}"
 	local -r virtiofs_log_prefix="virtiofs_"
 
+	local -r metrics_json_results_dir="${GOPATH}/${tests_repo}/metrics/results/"
+	local -r metrics_json_results_report_dir="${GOPATH}/${tests_repo}/metrics/results/new/"
```
Comment (chavafg) [inline .ci/teardown.sh:72]: `metrics_json_results_report_dir="${metrics_json_results_dir}/new"`?

File: .ci/teardown.sh [hunk 2]
```diff
@@ -310,6 +314,13 @@ check_log_files()
 		done
 	done
 
+	# Collect data generated by metrics CI
+	# If report was not geneated, data is created here
```
Comment (chavafg) [inline .ci/teardown.sh:318]: typo: generated

File: .ci/jenkins_job_build.sh [hunk 3]
```diff
@@ -60,6 +60,9 @@ init_ci_flags() {
 	# Check values for a profile defined as CLOUD
 	# Deprecated use METRICS_CI_PROFILE will be replaced by METRICS_CI_PROFILE=cloud-metrics
 	export METRICS_CI_CLOUD=""
+	# Genearte a report using a jenkins job data
```
Comment (GabyCT) [inline .ci/jenkins_job_build.sh:63]: s/Genearte/Generate

File: cmd/checkmetrics/history/generate_report_from_job.sh [hunk 4]
```diff
@@ -0,0 +1,86 @@
...
+set -o pipefail
+set -o errtrace
+
+script_name=${0##*/}
+script_dir=$(dirname "$(readlink -f "$0")")
+
+# Base dir of where we store the downloaded data.
+datadir="${script_dir}/data"
+
+
```
Comment (chavafg) [inline cmd/checkmetrics/history/generate_report_from_job.sh:21]: remove blank line?

File: .ci/teardown.sh (hunk 2 above)
Comment (GabyCT) [inline .ci/teardown.sh:318]: s/geneated/generated

File: cmd/checkmetrics/history/history.sh [hunk 5]
```diff
@@ -33,6 +33,8 @@ test_queries+=(".\"memory-footprint\".Results | .[] | .average.Result")
 tests+=("memory-footprint-ksm")
 test_queries+=(".\"memory-footprint-ksm\".Results | .[] | .average.Result")
 
+tests+=("memory-footprint-inside-container")
+
```
Comment (GabyCT) [inline cmd/checkmetrics/history/history.sh:37]: What about the test_query?

Comment (jcvenegas) [reply cmd/checkmetrics/history/history.sh:39]: Not needed, I only wanted for reports, added a comment to make it clear

**Pull request merged**